.travis.yml

```diff
@@ -1,22 +1,26 @@
+dist: trusty
+sudo: true
```

Comment:
[The docs][1] seem to say we want `sudo: required`. Any reason for the change to `true`?

[1]: https://docs.travis-ci.com/user/trusty-ci-environment